syntax = "proto3";

package student.academic.v1;

message RBPerson {
  string id = 1;
  string first_name = 2;
  string last_name = 3;
  string birth_date = 4;
  string birth_place = 5;
}
